atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -64,25 +64,21 @@ protected final <T, E extends Exception> T runTaskThrowOnConflict(TransactionTas
     protected final <T, E extends Exception> T runTaskThrowOnConflictWithCallback(
```

Comment:
NOW: can you basically get rid of this method now? presumably all the users should be able to be migrated to just a simple call to #onCommitOrAbort and we can unwind this sillyness? And finally everything can go through a single codepath, without this weird duplication between #runTaskThrowOnConflict and #runTaskThrowOnConflictWithCallback?